status.go

```diff
@@ -67,6 +67,44 @@ const (
 	versionError                  = "ERROR"
 )
 
+func (s *sequins) serveHealth(w http.ResponseWriter, r *http.Request) {
+	s.dbsLock.RLock()
+
+	status := status{DBs: make(map[string]dbStatus)}
+	for name, db := range s.dbs {
+		status.DBs[name] = copyDBStatus(db.status())
+	}
+
+	s.dbsLock.RUnlock()
+
+	ok := true
+	var statuses []string
+
+	// Iterate through all of the nodes of each database and ensure that
```

Comment:
Nodes as in computers in the database or block locally?  The health check should be about local health, not the state of the cluster as a whole.